src/back/TicketManager/src/Core/TicketManager.Core.Contracts/Common/IPaginatedRequest.cs

```diff
@@ -0,0 +1,7 @@
+namespace TicketManager.Core.Contracts.Common;
+
+public interface IPaginatedRequest
```

Comment:
Do we really want it to be interface? If we make it class we won't have to implement those two properties in every paginated request

Reply:
Ide implements it automatically anyway, so it's not a big deal. If we make it a class, we won't be able to make paginated request inheritance by any other class.